[AGM-1973] Create an article for obtaining trace logs.md (#166)
AdguardTeam/KnowledgeBaseVPN: docs/adguard-vpn-for-windows/solving-problems/logs.md

---

> title: How to collect and send logs

Reviewer: а кстати на русском это будет потом отдельно?

Author: перевод статьи? Да, эта задача только на англ была пока

Reviewer: на русском надо ставить отдельную или вы сами?

Author: создам задачу

Reviewer 2: зачем создавать задачу? у нас же переводы баз знаний в краудине

sidebar position: 1
---

If you encounter any problems while using AdGuard VPN for Windows, you can inform us about it. We would appreciate it if you also send application logs, as they help us resolve issues much quicker.

> ## Collecting and sending standard logs

Reviewer: у нас есть 3 главы `Collecting and sending %logsType% logs`, где последняя часть повторяется, т.е. прям копипаста явная. Может как-то выделить это в отдельную главу, а эти три сделать не `Collecting and sending ...`, а просто `Collecting...`

Author: а мне кажется, что удобно так. Например юзеру сказали, что надо отправить какие-то логи, он идёт в один раздел и сразу всё смотрит, не надо скакать по разным разделам, не?

Reviewer: ну я больше по програмистски - копипаста это зло. Если вы считаете, то так удобнее, то ок. Обсудите плз тогда внутри себя как лучше

1. Open AdGuard VPN for Windows and, if possible, repeat the actions that led to the error. Note the exact time when this error occurred.

1. Click **Support** (in the menu above) → **Report a bug**.

> ![Report a bug *border](https://cdn.adtidy.org/content/kb/vpn/windows/logs/support_report.jpeg)

Reviewer 3: надо локально собирать чтобы проверить как оно выглядит
https://uploads.adguard.org/sllxt88ofse0a.png

конкретно тут нужен отступ в 1 таб

Reviewer 3: и ниже тоже

1. In the opened form, enter your email address for feedback and describe the error found, including the time when this error occurred. If you can't reproduce the problem, specify as accurately as possible when it last occurred.

![Report *border](https://cdn.adtidy.org/content/kb/vpn/windows/logs/report_bug.png)

1. There is a check mark next to the **Send detailed system info**, which means that when you send a report, you also send logs.

> If for some reason it is more convenient for you to send us logs in another way, you can export them yourself. To do this, go to **Support** → **Export logs and system info**.

![Export *border](https://cdn.adtidy.org/content/kb/vpn/windows/logs/export.jpeg)

**Note**: if you encounter any problems while trying to export logs, there is an alternative way. Please follow these steps:

1. Open the Explorer app

> 1. Paste `%rpgramdata%/adguardvpn` in the search box

Reviewer: %programdata%

> 1. Find the folder called **Logs**

Author: не знаю точно почему, но знаю, что так надо)

> 1. Find the folder called **Logs**

Reviewer: ок

> 1. Find the folder called **Logs**

Reviewer 2: список всё равно будет отрисвываться как 1, 2, 3, но если мы решим добавить/убрать пункты, это повлечёт за собой меньше правок. что-то такое @slavaleleka как-то рассказывал

1. In the **Logging level** section, select **Record everything**.

![Logging level *border](https://cdn.adtidy.org/content/kb/vpn/windows/logs/logging.jpeg)

1. Repeat the actions that led to the error, and note the time when it occurred.

1. Switch the logging level back to **Record by default**.

1. Then select **Support** (in the menu above) → **Report a bug**.

![Report *border](https://cdn.adtidy.org/content/kb/vpn/windows/logs/report_bug.png)

1. In the opened form, enter your email address for feedback and describe the error found, including the time when this error occurred.

1. Make sure that there is a check mark next to **Send detailed system info** and click **Send**.

> If for some reason it is more convenient for you to send us logs in another way, you can export them yourself. To do this, go to **Support** → **Export logs and system info**.

![Export *border](https://cdn.adtidy.org/content/kb/vpn/windows/logs/export.jpeg)

## Collecting and sending trace logs

Sometimes our support team might ask you to send the trace logs. To do this, you need to do the following steps:

> 1. Right click the AdGuard VPN icon in the trace menu and choose *Exit AdGuard VPN*.

Reviewer: trace menu -> tray menu

![Exit *mobile](https://cdn.adtidy.org/content/kb/vpn/windows/logs/exit.png)

> In the opened window, agree to close the service

Reviewer: Это окно появится только если пользователь приконнекчен. Лучше написать "если появится диалог (в частности он появится, если установлено соединение с ендпоинтом), то нажмите "ок"

![Attention *mobile](https://cdn.adtidy.org/content/kb/vpn/windows/logs/attention.png)

1. Open the console (type cmd in the start panel).

1. Run the application with the command `C:\"Program Files (x86)"\AdGuardVpn\AdGuardVpnSvc.exe /trace` if you are using 64-bit Windows, and `C:\"Program Files"\AdGuardVpn\AdGuardVpnSvc.exe /trace` if you are using 32-bit.

1. Reproduce the issue. We strongly recommend that you note the exact time when you reproduce the problem: this will help our support team to find relevant log entries and resolve the problem faster.

1. Export the recorded logs: *Support* -> *Export logs and system info*.

Don't worry if your log file is large. We need to know as many details as possible in order to fix your problem.

> **Please note**: if you encounter some problems while trying to export logs, there is an alternative way. Please follow these steps:

Reviewer: сейчас выглядит так, как будто этим способом можно воспользоваться только для трейсовых логов - это конечно не так и было бы лучше указать на эту возможность также после [аналогичного блока](https://github.com/AdguardTeam/KnowledgeBaseVPN/pull/166/files#diff-fea39d891fe7293bee9ba427229b74d675567785d0c6dd7b8ab9efe7b75d3d58R50) в общем разделе

> 1. Open *My PC* and go to *ProgramData* -> *AdguardVPN* folder in Local Disc (C:)

Reviewer: всё же как и мы общались в переписке - сложно будет объяснить, что такое *ProgramData* -> *AdguardVPN* folder in Local Disc (C:), лучше по шагам
1. open `explorer` app
2. paste `%rpgramdata%/adguardvpn` in the search box
3. ...

![Export *border](https://cdn.adtidy.org/content/kb/vpn/windows/logs/export.jpeg)

> **Please note**: if you encounter some problems while trying to export logs, there is an alternative way. Please follow these steps:

Reviewer 2: Note: if you encounter any problems while trying to export logs, there is an alternative way. Please follow these steps:

Reviewer 2: можно и без ноут, мы обычно адмонишн вместо него ставим, но тут ниже инструкция, не очень подойдёт

> 1. Open `explorer` app

Reviewer 2: Open the Explorer app

1. Paste `%rpgramdata%/adguardvpn` in the search box

> 1. Locate the folder called **Logs**

Reviewer 2: Find?

2. In the **Logging level** section, select **Record everything**.

![Logging level *border](https://cdn.adtidy.org/content/kb/vpn/windows/logs/logging.jpeg)

3. Repeat the actions that led to the error, and note the time when it occurred.

4. Switch the logging level back to **Record by default**.

5. Then select **Support** (in the menu above) → **Report a bug**.

![Report *border](https://cdn.adtidy.org/content/kb/vpn/windows/logs/report_bug.png)

6. In the opened form, enter your email address for feedback and describe the error found, including the time when this error occurred.

7. Make sure that there is a check mark next to **Send detailed system info** and click **Send**.

> >If for some reason it is more convenient for you to send us logs in another way, you can export them yourself. To do this, go to **Support** → **Export logs and system info**.

Reviewer 2: и не хватает пустой строки перед ![Export *border](https://cdn.adtidy.org/content/kb/vpn/windows/logs/export.jpeg)

Don't worry if your log file is large. We need to know as many details as possible in order to fix your problem.

## Sending logs

To send logs to our support team, please follow the next steps:

> 1. Click **Support** (in the menu above) → **Report a bug**.

Reviewer 2: скобки лишние

![Report a bug *border](https://cdn.adtidy.org/content/kb/vpn/windows/logs/support_report.jpeg)

> 1. In the opened form enter your email address for feedback and describe the error found, including the time when this error occurred. If you can't reproduce the problem, specify as accurately as possible when it last occurred.

Reviewer 2: ```suggestion
1. In the form that opens, enter your email address and describe the error you found, including the time it occurred. If you can't reproduce the problem, be as specific as possible about when it last occurred.
```

> If for some reason it is more convenient for you to send us logs in another way, you can export them yourself. To do this, go to **Support** → **Export logs and system info**.

![Export *border](https://cdn.adtidy.org/content/kb/vpn/windows/logs/export.jpeg)

> **Please note**: if you encounter some problems while trying to export logs, there is an alternative way. Please follow these steps:

Reviewer 2: тот же коммент, что и выше

> 1. Open `explorer` app

Reviewer 2: и тут

1. Paste `%rpgramdata%/adguardvpn` in the search box

> 1. Locate the folder called **Logs**

Reviewer 2: и тут

1. Archive the folder

1. Send it to our support team

## Collecting and sending trace logs

> Sometimes our support team might ask you to send the trace logs. To do this, you need to do the following steps:

Reviewer 2: ```suggestion
Sometimes our support team might ask you to send them trace logs. To do this, you need to do follow these steps:
```

> 1. Right click the AdGuard VPN icon in the tray menu and choose **Exit AdGuard VPN**.

Reviewer 2: Right-click

![Exit *mobile](https://cdn.adtidy.org/content/kb/vpn/windows/logs/exit.png)

> Agree to close the service if you see the warning message.

Reviewer 2: ```suggestion
If you see the warning message, agree to stop the service.
```
скорее так

![Attention *mobile](https://cdn.adtidy.org/content/kb/vpn/windows/logs/attention.png)

1. Open the console (type `cmd` in the start panel).

1. Run the application with the command `C:\"Program Files (x86)"\AdGuardVpn\AdGuardVpnSvc.exe /trace` if you are using 64-bit Windows, and `C:\"Program Files"\AdGuardVpn\AdGuardVpnSvc.exe /trace` if you are using 32-bit.

1. Reproduce the issue. We strongly recommend that you note the exact time when you reproduce the problem: this will help our support team to find relevant log entries and resolve the problem faster.

> 1. Export the recorded logs: **Support** -> **Export logs and system info**.

Reviewer 2: →

Don't worry if your log file is large. We need to know as many details as possible in order to fix your problem.

> **Please note**: if you encounter some problems while trying to export logs, there is an alternative way. Please follow these steps:

Reviewer 2: и тут тот же коммент

> 1. Open `explorer` app

Reviewer 2: и тут

1. Paste `%rpgramdata%/adguardvpn` in the search box

> 1. Locate the folder called **Logs**

Reviewer 2: и тут
